Library/Homebrew/keg.rb

```diff
@@ -81,23 +81,24 @@ def to_s
       opt
     ]).map { |dir| HOMEBREW_PREFIX/dir }
   ).freeze
+  PRESERVED_DIRECTORIES = (%w[
```

Comment:
Do we need an extra variable here? Can the `MUST_BE_WRITABLE_DIRECTORIES` or `MUST_EXIST_DIRECTORIES` be used instead?

Reply:
At least on my machine, `MUST_BE_WRITABLE_DIRECTORIES` includes `lib/python3.7` and company. If we're okay with preserving those too, we can definitely just use that. `MUST_EXIST_DIRECTORIES` only has `bin etc include lib sbin share var Frameworks opt`; would the subdirectories that we're trying to preserve (e.g. `share/man/man[1-8]`) be kept?

Reply:
@alebcay Let's use `MUST_BE_WRITABLE_DIRECTORIES` 👍. Apologies for earlier confusion.